### PR TITLE
swift 3 and 4 compatibility 

### LINKED ISSUE #235: Any support for Swift 4 ?

### PR DESCRIPTION
**Pull request by lgaches**
fix #235

### CODE REVIEW

**Comment by skreutzberger** [comment]
@lgaches awesome, thanks a lot 💯❤️ ! As always, I will ping you on Slack for details.

File: Sources/AES256CBC.swift [hunk 1]
```diff
@@ -822,6 +786,31 @@ fileprivate extension Collection where Self.Iterator.Element == UInt8, Self.Inde
         }
         return result
     }
+    #else
+    func toInteger<T: Integer>() -> T where T: ByteConvertible, T: BitshiftOperationsType {
```
Comment (jshier) [inline Sources/AES256CBC.swift]: Indentation broken down here.

**Comment by dunqi** [comment]
Good work 😉 thank you for the fast fix

**Comment by eman6576** [comment]
Can this be merged?

**Comment by Narsail** [comment]
Would be nice if it is merged soon.

**Comment by irace** [comment]
Please merge 🙏

**Comment by zdenektopic** [comment]
yes, please do 👍 🙏

**Comment by skreutzberger** [comment]
Laurent's pull request which brought official support for Swift 3.2 and Swift 4 was released today as SwiftyBeaver 1.4.0. Thanks again for the contribution & testing 🎉

**Pull request merged**